connectors/airwatch/README.md

```diff
@@ -39,5 +39,33 @@ docker run --name airwatch-connector \
            ws1connectors/airwatch-connector \
            --spring.config.additional-location=file:/mnt/managed-apps.yml \
            --server.port=8080 \
+           --greenbox.url="https://acme.vmwareidentity.com" \
            --security.oauth2.resource.jwt.key-uri="https://acme.vmwareidentity.com/SAAS/API/1.0/REST/auth/token?attribute=publicKey&format=pem"
 ```
+
+## RPM
+
+In general connectors are built, installed and run as RPMs as explained in [README](https://github.com/vmware/connectors-workspace-one/blob/master/README.md#rpm).
+Same guide should be followed but make sure to supply the additional configurations required for the AirWatch App Discovery Connector, before starting the service.
+Additional configurations include details of manged apps and Greenbox url.
+
+Assuming you have a `managed-apps.yml` in `/my/config/dir`:
+
+Provide a copy of the file for the service to read.
+```
+cp /my/config/dir/managed-apps.yml /etc/opt/vmware/connectors/airwatch/managed-apps.yml
```

Comment:
Also add in a `chown roswell:roswell /etc/opt/vmware/connectors/airwatch/managed-apps.yml`.  If they are running commands as root, then it would produce a file owned by root that maybe the aw connector could not read.